data/human/post-war reactions.txt

```diff
@@ -0,0 +1,257 @@
+				goto questions
+			label young
+			`	As you start walking, he answers. "No, I'm an orphan." He sees your confusion and explains. "The Syndicate lets some of us orphans work in industry. In return they give us better schooling, nicer housing, and doctor's visits." He puffs himself up. "Only the cleverest of us get to do it. All of the jobs for orphans are safe too; we don't use big dangerous equipment. Just ordinary stuff, like hammers, soldering irons, or sewing machines.`
+			`	"Wait a minute!" He suddenly looks at you with wide eyes. "You're a captain, right? Were you in the big fights? I was on Maker when I saw some Pug ships fighting our ships on our school's big edu-screen."`
+			choice
+				`	"Yes, I'm Captain <first> <last>."`
+					goto me
+				`	"I blew up my share of Pug ships."`
+					goto blew
+				`	"Actually, I led the battle."`
+			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You're Captain <first> <last>! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand, and you shake it.`
+				goto next
+			label me
+			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You led the attack! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand, and you shake it.`
+				goto next
+			label blew
+			`	"I bet you did! I wish I was on one of our warships, bravely defending us all. Blowing up Pugs, getting paid. It sounds amazing!"`
+			label next
+			`	"What was it like, fighting those nasty, evil Pugs?"`
+
+			choice
+				`	"Actually, they're not evil. Their 'war' was a ploy to get us to stop fighting among ourselves. A lot of Pug gave their lives willingly to save us from self-destruction."`
+				`	"War is hell, kid. I'm not going to glorify it for you."`
+					goto hell
+				`	"Leading a battle is like chess with guns and engines. It's the most exhilarating feeling I've ever felt."`
+					goto chess
+				`	"It was an honor to protect the human race from our attackers."`
+					goto honor
+			`	"Woah, you sound really smart. Like one of those sensible experts in the vids! Still, without people like you, the Pug's plan couldn't have worked. I still reckon you were the best, Captain."`
+				goto salute
+			label hell
+			`	"You went through hell for us, eh? That sounds less fun than I thought... Maybe I should stick to school and be an engineer instead of a pilot. I dunno."`
+				goto salute
+			label chess
+			`	"The Syndicate is training me to maintain and they're even going to train me to pilot their combat ships when I'm older! One day, I'll fight aliens like you did."`
+				goto salute
+			label honor
+			`	"Honor, and service. The Syndicate will train boys like me, but I've wondered about joining the Navy. It isn't as profitable, but it sounds more noble."`
```

Comment:
```suggestion
			`	"Honor... and service. The Syndicate will train boys like me, but I've wondered about joining the Navy. It isn't as profitable, but it sounds more noble."`
```

If the point is to have a pause in the kid's speech here, I think this route will be better.

If it's not, just cut the extra comma.